lib/widgets/components/card.dart

```diff
@@ -58,15 +54,22 @@ abstract class CardBase extends StatelessWidget {
     );
   }
 
+  void Function()? _onTap(BuildContext context) {
+    if (onTap == null) return null;
+    return () {
+      onTap!(context);
+    };
+  }
+
   @override
   Widget build(BuildContext context) {
     return Tappable(
-      elevation: 1,
+      elevation: onTap == null ? 0 : 1,
       padding: EdgeInsets.all(dense ? 16 : 24),
       color: color,
       borderRadius: BorderRadius.circular(24),
       borderColor: borderColor,
-      onTap: onTap,
+      onTap: _onTap(context),
```

Comment:
Instead of having the insternal _onTap function, we could replace it with a ternary on this line like so
`onTap != null ? () => onTap!(context) : null`
What do you think?